### ISSUE TITLE
Configure accessibility insight tool to CI/CD pipeline

### ISSUE THREAD

**Issue by adhiambovivian** [microsoftgraph/microsoft-graph-explorer-v4#2466]

**Comment by Onokaev** [comment]
Closing this as the playwright accessibility tests already test for the required accessibility violations
